secret_programmatic_api_keys.go

```diff
@@ -249,6 +275,23 @@ func (b *Backend) pathProgrammaticAPIKeyRollback(ctx context.Context, req *logic
 	}
 
 	if isProjectKey(entry.OrganizationID, entry.ProjectID) {
+
+		// we need the orgID to delete the Key
+		foundKey := mongodbatlas.APIKey{}
+		keys, _, err := client.ProjectAPIKeys.List(ctx, entry.ProjectID, nil)
+		if err != nil {
+			return err
+		}
+		for _, key := range keys {
+			if key.ID == entry.ProgrammaticAPIKeyID {
+				foundKey = key
+				break
+			}
+
+		}
+
+		orgID := foundKey.Roles[0].OrgID
```

Comment:
Won't this panic if the key wasn't found in the `for` loop? `Roles` will be nil.